tutorials/Window.getCurrent.md

```diff
@@ -0,0 +1,18 @@
+Returns an instance of the current window.
+# Example
+```js
+async function getCurrent() {
+    const app = await fin.Application.create({
+        name: 'myApp',
+        uuid: 'app-1',
+        url: 'https://www.openfin.co',
+        autoShow: true
+    });
+    await app.run();
+    await app.getWindow();
+    const win = await app.getCurrent();
```

Comment:
This is a tutorial on how to use the app.getCurrent. Lets get something similar to:

http://cdn.openfin.co/jsdocs/stable/tutorial-window.getCurrent.html

And make a note that it will only work in openfin.